pydantic/fields.py

```diff
@@ -441,11 +453,16 @@ def Field(
         if not json_schema_extra:
             json_schema_extra = extra
 
+    if serialization_alias is None and isinstance(alias, str):
+        serialization_alias = alias
+
     return FieldInfo.from_field(
         default,
         default_factory=default_factory,
         alias=alias,
         alias_priority=alias_priority,
+        validation_alias=validation_alias or alias,
```

Comment:
```suggestion
        validation_alias=validation_alias if validation_alias is not None else alias,
```
Could also just do this the same way you handle `serialization_alias` above, or else you can't do `validation_alias = ''`. I believe there are some tests for `alias = ''`, as weird a case as that is.